vm/src/stdlib/subprocess.rs

```diff
@@ -142,8 +142,8 @@ impl PopenRef {
         self.process.borrow().exit_status()
     }
 
-    fn wait(self, timeout: OptionalArg<u64>, vm: &VirtualMachine) -> PyResult<()> {
-        let timeout = match timeout.into_option() {
+    fn wait(self, timeout: OptionalArg<Option<u64>>, vm: &VirtualMachine) -> PyResult<()> {
```

Comment:
Could you add a `PopenWaitArgs` struct so that the `timeout=` kwarg works?

Reply:
done